python/tests/test_highlevel.py

```diff
@@ -2991,17 +2971,18 @@ def test_metadata(self):
         (inst,) = self.get_instances(1)
         (inst2,) = self.get_instances(1)
         assert inst == inst2
-        inst._metadata_decoder = lambda m: "different decoder"
+        inst.metadata
```

Comment:
What does this statement do?

Reply:
I thought it was forcing decode of the metadata, but that is done by the equality operator above anyway, removed.